docs/interact-with-code.ipynb

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
you can't really interact with code here. Maybe a better title for the notebook and section would be "View Rendered Notebooks"